vortexasdk/endpoints/fixtures.py

```diff
@@ -0,0 +1,176 @@
+"""
+from vortexasdk.endpoints.fixtures_result import FixtureResult
+from vortexasdk.endpoints.endpoints import FIXTURES
+from vortexasdk.operations import Search
+from vortexasdk.api.shared_types import to_ISODate
+from datetime import datetime
+from typing import Any, Dict, List, Union
+from vortexasdk.api import ID
+from vortexasdk.utils import convert_to_list
+
+
+class Fixtures(Search):
+    """Fixtures Endpoint, use this to search through Vortexa's Fixtures data.
+
+    A detailed explanation of Cargo/Vessel Movements can be found [here](https://docs.vortexa.com/reference/intro-fixtures).
+
+    """
+
+    _MAX_PAGE_RESULT_SIZE = 500
+
+    def __init__(self):
+        Search.__init__(self, FIXTURES)
+
+    def search(
+        self,
+        filter_time_field: str = "fixing_timestamp",
+        filter_time_min: datetime = datetime(2020, 1, 1),
+        filter_time_max: datetime = datetime(2020, 1, 2),
+        ids: Union[ID, List[ID]] = None,
+        filter_charterers: Union[ID, List[ID]] = None,
+        filter_destinations: Union[ID, List[ID]] = None,
+        filter_origins: Union[ID, List[ID]] = None,
+        filter_owners: Union[ID, List[ID]] = None,
+        filter_effective_controllers: Union[ID, List[ID]] = None,
+        filter_products: Union[ID, List[ID]] = None,
+        filter_vessels: Union[ID, List[ID]] = None,
+        filter_vessel_classes: Union[str, List[str]] = None,
+        filter_vessel_age_min: int = None,
+        filter_vessel_age_max: int = None,
+        filter_vessel_scrubbers: str = "disabled",
+        filter_vessel_flags: Union[ID, List[ID]] = None,
+        exclude_origins: Union[ID, List[ID]] = None,
+        exclude_destinations: Union[ID, List[ID]] = None,
+        exclude_products: Union[ID, List[ID]] = None,
+        exclude_vessels: Union[ID, List[ID]] = None,
+        exclude_vessel_classes: Union[str, List[str]] = None,
+        exclude_charterers: Union[ID, List[ID]] = None,
+        exclude_vessel_flags: Union[ID, List[ID]] = None,
+        order: str = None,
+        order_direction: str = None,
+    ) -> FixtureResult:
+        """
+        Find Fixtures for a given preset and date range.
```

Comment:
What do you mean by "given preset"? Is this copied from a different endpoint? There are some endpoints that work by presets, but I don't think fixtures is one of them.